src/app/models/form.server.model.ts

```diff
@@ -408,6 +436,18 @@ const compileFormModel = (db: Mongoose): IFormModel => {
     return { ...newForm, ...overrideProps }
   }
 
+  FormSchema.methods.getPublicView = async function (this: IFormSchema) {
+    // Populate form if not populated yet.
+    if (!this.populated('admin')) {
+      await this.populate(POPULATE_FORM_PARAMS).execPopulate()
+    }
+
+    return {
+      ...(pick(this, FORM_PUBLIC_FIELDS) as PublicFormValues),
+      admin: pick(this.admin, 'agency'),
+    }
```

Comment:
seems like which keys to show are defined both in `POPULATE_FORM_PARAMS` and over here using lodash's `pick`. would it be possible to only define them above in one place, then merely call `return this.toObject()` here?

Reply:
(do take the above under advisement given other conflicting suggestions)